### ISSUE TITLE
Accessibility on DUP Public -> Missing option to skip to main content

### ISSUE THREAD

**Issue by manuji** [bcgov/parks-reso-public#501]
**Describe the bug**
According to Success Criterion 2.4.1 Bypass Blocks in WCAG 2.2 AA, there should be a mechanism to bypass repeated content blocks. On DUP Public, there is no option to "Skip to main content" to bypass the header bar.  

**To Reproduce**
1. Go to '[DUP Public website](https://reserve.bcparks.ca/dayuse/)'
2. Using the keyboard tab key, try to navigate through the wbsite
3. Check if you can directly navigate to the main content of the website rather than navigating through the header bar content (such as site logo & etc..)

**Expected behavior**
Should give the user an option to "Skip to main content".

**Comment by Christopher-walsh22** [comment]
I think for this ticket we will need a design. From reading up on what these buttons are supposed to accomplish, It will need to be the first button on every page (before our bcparks logo) and appear on every page. 

@LindsayMacfarlane  @kcabhar @pavelcreates let me know what you think

**Comment by manuji** [comment]
Hi @Christopher-walsh22 , it should not appear on the UI. It should only appear when you start tabbing. On the first tab key the button or link will be displayed. There is no need of a design input for that. 

Refer to bcparks.ca and new york times for examples.

https://github.com/user-attachments/assets/39b2329b-8ae5-4814-897a-5609f697104f

**Comment by Christopher-walsh22** [comment]
The examples I was looking at were different, Thanks for the context Manuji!

**Comment by manuji** [comment]
Tested on TEST: Passed
* verified that this issue has been fixed. 
* Moving to Done. 

Note: On pages except the home page, the link is not displayed when you first hot tab, but it's not necessary as it tabs in to the main content anyways.

**Comment by manuji** [comment]
Deployed to PROD May 11, 2026
